classes/04class/exercises/c04-iac02/carloshz4/output.tf

```diff
@@ -0,0 +1,20 @@
+output "vpc_cidr" {
+  value = "${aws_vpc.devopsacademy-iac.cidr_block}"
+}
+
+output "subnet1_cidr" {
+  value = "${aws_subnet.subnet1.cidr_block}"
```

Comment:
The syntax with `"${}"` when not concatenating a string is deprecated.

Just use the value directly: `aws_subnet.subnet1.cidr_block`